ehrapy/tools/_sa.py

```diff
@@ -116,15 +116,10 @@ def glm(
 
 
 def kmf(
-    durations: Iterable,
-    event_observed: Iterable | None = None,
-    timeline: Iterable = None,
-    entry: Iterable | None = None,
-    label: str | None = None,
-    alpha: float | None = None,
-    ci_labels: tuple[str, str] = None,
-    weights: Iterable | None = None,
-    censoring: Literal["right", "left"] = None,
```

Comment:
add comment that and "right" just calls fit and "left" calls `fit_left_censoring`, maybe change default value to "right" instead of None